libsql-server/src/namespace/meta_store.rs

```diff
@@ -475,11 +609,13 @@ impl MetaStoreHandle {
                 c.borrow_and_update();
                 let changed = c.changed();
 
+                let (snd, rcv) = oneshot::channel();
                 changes_tx
-                    .send((self.namespace.clone(), new_config))
+                    .send((self.namespace.clone(), new_config, snd))
                     .await
                     .map_err(|e| Error::MetaStoreUpdateFailure(e.into()))?;
 
+                rcv.await??;
                 changed
```

Comment:
Can we get rid of DatabaseConfig::changed() now that we have channel as part of a ChangeMsg?